server/bin/gold/test-7.10.txt

```diff
@@ -11257,5 +11257,5 @@ run-1970-01-01T00:00:42-UTC: Processed 1 tarballs
c4a54479982996eb15ea9f", "279f1458a1e4e00bda809c93f067e3cb", "6522af48f616f9c71f992143cffd57b4", "01b3e483e32a12628e151b9d78ee74ab", "87bbdfb9df1bb13c1d4d6a97d8ab498e", "3150f2537a76fbb07b99fb3e678767ec", "65588cd78acb93c2251c9dd091f5b244", "9c60e58c3600ec695d60263ea497b0e2", "cf3cb52a7f31fb2ca96bb20d74de12e4", "3309dcee7e2470b0c63c41932ceb1827", "9f86adab0beff52ce7bb33cb4b0440d2", "df0005730d8108cf25cee77f3c6fc77a", "02c8241452633513da8cab330db21e20", "cb6cd502b4120fd195af56f5f1fc6078", "d35a03a4460f1d2bb8b47a085da53aeb", "0794991e4effd5338c2e3bd5d469d014", "b9896b5cde72b5ec2ecf248508e18ea7", "2449df7807a74c69d73c56bbf83ee3b4", "a8150c571f2c8d5f0fb5b88d0d5e3648", "f75a9132dbf337a78e171453905ea237", "bb753a16295aa236ec70d97b574e01ed", "405fad2ec88416e33602949793828941", "7f1271b5fa6153bb6822f17006e563c3", "78191a9754f4ec2a0e12a1bedf1f412d", "08fb0ca05f32579d16fda103d944fe96", "75d1066c03fbfaee537ae9ffe9f48b8f", "515f19003d9beacbea37194a96cc2daa", "57cd38d1bbce67d011d52bde0c84c278", "5793b2279e80d46e68008b6607ece249", "c9b7520669e16c13e216a187c3b804c5", "556eff06d022c29e5641059a1d7484a3", "cdd213e66faed367e356497b8c9e1338", "37afd655752d6cb13bb2e47560156fe0", "71b67fc3c581a542dfb9e340149fd563", "5aa2a32b653b23cafdae543ba70f134e", "95e0b15d6dc4bc1ffef73a9decf32238", "715424108c6c1368c9b18ddac8b4703a", "c983225bd9bfdd82b5cedc2a58505ecb", "c0719f6fbba807220ecc2f1a261ad357", "25951304e91d40032728eb418d8aab5b"], "pbench-unittests.v4.tool-data-proc-vmstat.2018-02-02": ["da9973c96a6eb7c26bbdf1220faaa432", "df6f60c7c3587f6093218a411cdc8032", "54c3ee7bffb4ba6ef67f0fd160eec656", "31ae608ccf8d1c024aa86698d0256e3f", "896a17abca95aa0529330177f0884dab", "b7c6c55684e5ec9dabb534686cbf763f", "ca8c1dad53cae448c4b3bbaafd63a667", "ee7d78d8293220c9886dd8c954411abd", "5ac09a0a7e3977aeaa9204b18bdf616c", "2d6dc7d7d0f57a400af37a9be7bb4962", "bb5dea9f650a321856587379f18b0770", "3160511681eeb73c0bfeb228153779d2", "b652944f7e078df588e8bcc9ad7fa99b", "7ed6bdc342b46131baaa383a6babff17", "5207bfd5b0e8012a26364a3e733f63c3", "d31e711e51abee001893f86b09e3fbb4", "955f3fc22edef6b177572e9452bea881", "49f297d07bbe51f1f8def65828a75728", "1947fa8d7afad8b3e7a8a17ea50b414f", "052df22fc918425a804ca9f79dbab2dc", "e81f4d9505670a66c4f7e3e86ccf75c7", "1dbf2c4b42c66d47657d33600d3765da", "b51c0aa02763406e3580721847b51e14", "03f7ec6140440f8baaa4d1906ddf5e7a", "50e2c1d85e04c38150d8e65f0ef92f33", "1b0fd18e66c3369b5d6689dd3e361e36", "c3b2b2aad62725017ecd1bd9ca2d8227", "ce3a86c687f3eb1dba6fd7476a4106fe", "0ecfb045e0961c2dd3eda40134dbf164", "46c7e28a54e9ac271cfff604abfad753", "4e5ae31695b9c6c38ddceece8c5e278f", "d947d04f42fb1c658d4d4929c412b8a0", "865f631a24e97f8b3727ae6d5c3bad65", "dc4d59e3feb08c8c7e933f2d45c9057c", "eb33fcaea8eec9b0181b8a4b62512622", "35f351825af6cb447829e4f28127ef96", "f21f6433231c2ae377a95ccc0c35580d", "f78c7c6d3a593da7bfd518608eae98c8", "278c2482da8ade228bb098c89e64321d", "03bdca4f5d8f8f5b75d1df4239f3a74c", "fe6b27d44993167bd94f5b31b03ea4e9", "234387ba670fce0abd6a4b10353a454c"]}}
```

Comment:
Yeah, it's a long map, because there's tool data.

And the difference ... well ... this is because I accidentally fixed a bug that had gone unnoticed before. I'd carefully merged the tool data index pass map into the primary map and updated the DB ... but ... remember that earlier comment about the `.copy()`??? Yeah ... before I was updating the map with the new data, but SQLAlchemy didn't update the DB because the top level resource pointer hadn't changed. Oopsies.

This is now correct.

Reply:
And this is a *big* chunk of cruft in the conversation tab! I'm going to resolve the conversation to get it hidden. I my thumb doesn't get tired scrolling back to the previous comment. 🤣